olm-catalog/serverless-operator/project.yaml

```diff
@@ -21,7 +21,7 @@ requirements:
     # is ignored as it is overridden by fake version via KUBERNETES_MIN_VERSION.
     # This value is used for CSV's min version validation.
     minVersion: 1.21.0
-  golang: '1.18'
+  golang: '1.19'
```

Comment:
I wonder if the property is useful anywhere though.

Reply:
Yes it is used to specify the verison in `openshift/ci-operator/build-image/Dockerfile`. See https://github.com/openshift-knative/serverless-operator/blob/2157cc9bed970860264b22649a78be4416024ebd/hack/generate/dockerfile.sh#L20 and https://github.com/openshift-knative/serverless-operator/blob/506f33152372b36703304f103894e99231fbfd63/templates/build-image.Dockerfile#L3

Reply:
Thanks, for the pointer that's a interesting coupling. I don't see any other use of the variable. Because we probably want to control all of the Go build versions at once.

Reply:
Yeah not ideal. It's quite confusing at how may places we have to update the version :)